src/store/metadata/stub/content/editions/versions/pupils-at-expected-standards.json

```diff
@@ -0,0 +1,88 @@
+            "download_url": "https://staging.idpd.uk/datasets/pupils-at-expected-standards/editions/2023-03/versions/1.csv",
+            "media_type": "text/csv",
+            "table_schema": {
+                "columns": [
+                    {
+                      "name": "period",
+                      "datatype": "string",
+                      "titles": "Period",
+                      "description": "The time period associated with the observation."
+                    },
+                    {
+                      "name": "areacd",
+                      "datatype": "string",
+                      "titles": "ONS Geography Code",
+                      "description": "The geography code associated with the observation."
+                    },
+                    {
+                      "name": "areanm",
+                      "datatype": "string",
+                      "titles": "Area Name",
+                      "description": "The name assiciated with The geography code "
+                    },
+                    {
+                      "name": "geography",
+                      "datatype": "string",
+                      "titles": "Geography Type",
+                      "description": "The categorisation of the geography code"
+                    },
+                    {
+                      "name": "value",
+                      "datatype": "decimal",
+                      "titles": "Geographic area with percentage of pulpils at expected standards by at least one provider",
+                      "description": "Observational Value"
+                    },
+                    {
+                      "name": "measure",
+                      "datatype": "string",
+                      "titles": "Measure Type",
+                      "description": "Measure Type of the Observational Value"
+                    },
+                    {
+                      "name": "unit",
+                      "datatype": "string",
+                      "titles": "Unit",
+                      "description": "Unit Type of the Observational Value"
+                    },
+                    {
+                      "name": "lower confidence interval (95%)",
+                      "datatype": "decimal",
+                      "titles": "Lower Confidence Interval",
+                      "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Lower bound"
+                    },
+                    {
+                      "name": "upper confidence interval (95%)",
+                      "datatype": "decimal",
+                      "titles": "Upper Confidence Interval",
+                      "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Upper bound"
+                    },
+                    {
+                      "name": "observation status",
```

Comment:
obeservation_status, names should be lowercase and spaces replaces with an underscore.